tcms_tenants/templates/tcms_tenants/new.html

```diff
@@ -41,6 +42,11 @@
                                class="form-control" required
                                {% if form.schema_name.field.disabled %}disabled{% endif %}>
                         <span class="input-group-addon">.{{ tcms_tenants_domain }}</span>
+                        <span class="input-group-addon">
+                            {% if object %}
+                                {% attachment_form object%}
+                            {% endif %}
+                        </span>
```

Comment:
Location of this widget is not good. It messes up the display of the rest of the widgets, see
![image](https://user-images.githubusercontent.com/1002300/193527496-0de34cb0-3860-4556-acef-ff41cc7b748e.png)

Place it as the last widget before the Save button. It should have a label of "Logo"

Reply:
Clicking the Save button regardless of whether or not I have a file chosen doesn't seem to do anything.

Reply:
Also why not use `tcms/templates/include/attachments.html` like everywhere else ?